README.md

```diff
@@ -1,347 +1,514 @@
+5.  . Additional branching is allowed but merge all changes into the branch tied to the issue.
+6.  Add your changes to the code.
+    -   The command `git status` shows which files have changed.
+
+    -   `git diff [filename]` will highlight which lines have been modified.
+
+    -   Use the arrow keys to scroll, and press `q` if you need to exit the prompt.
 
-**Please reach out Aaron with any questions.** The only thing he loves more than version control is helping people learn version control. 
+    -   `git add [filename]` will stage files to commit (`git add -u` will add all modified files).
+
+    -   `git commit -m "[your message here]"` will commit changes to version control.
+        Commit messages should be clear and meaningful.
+
+    -   git push origin [issue branch name] will push committed changes up to the GitHub branch.
+7.  To contribute to the `main` branch, put in a Pull Request. Tag your assigned reviewer (@reviewer). Briefly describe what the PR does.
+8.  Aaron R. Williams will review and incorporate changes into the `main` branch. He may ask you to make changes. **For Urban employees only, please reach out to the "umf-mobilitymetrics3" slack channel if you have questions.**
 
 # Data Standards
 
-* All final files should be in the `.csv` format. The files should be delimited with a comma. 
-* Files should have descriptive names about the metrics and only include lower case letters, numbers, and underscores (lower camel case, i.e. camel_case). Do not use spaces.
-* Do not open and save any `.csv` files in Microsoft Excel. Excel defaults are not sensible and lead to analytic errors. 
-* Starting data should be pulled using code or should be added to Box. Final data files should be added GitHub repository.
-* FIPS codes should always contain leading zeros so that state codes are two digits, county codes are three digits, and place codes are five digits. 
+This section will walk through the standards around data starting with the raw data used to create the metric, joining variables that need to be included in every file, data naming and sorting conventions, data quality standards and standard errors, subgroup files and file naming/final metric file standards.
+
+### Starting or raw data practices
+
+-   Starting or raw data should be pulled using code or should be added to Box. Only final data files should be added GitHub repository.
+-   Whenever possible, programs should pull data directly from the original source, meaning raw data does not have to be downloaded manually prior to code execution for the program to work. For example, programs that rely on American Community Survey (ACS) data should write code that utilizes the [Census API](https://www.census.gov/data/developers/data-sets.html) to pull in micro data; for [R programmers] the [ipumsr package] ( <https://developer.ipums.org/docs/v2/apiprogram/clients/>) is a great tool for pulling ACS micro data from the Census API using code.
```

Comment:
The hyperlinks here are busted because of the formatting: [Census API](https://www.census.gov/data/developers/data-sets.html) to pull in micro data; for [R programmers] the [ipumsr package] ( <https://developer.ipums.org/docs/v2/apiprogram/clients/>)